examples/sycl/ls-sycl-device.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * #include "common.h"
+
+#include "console.h"
+#include <cassert>
+#include <cinttypes>
+#include <cmath>
+#include <cstdio>
+#include <cstring>
+#include <ctime>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+*/
+
```

Comment:
Should all of this be commented out? I think it should either be removed or uncommented

Reply:
yes, clear it.